public/json/open_terminal_window.json

```
@@ -0,0 +1,31 @@
+{
+  "title": "Open a Terminal window with Ctrl-RAlt-T",
+  "rules": [
+    {
+        "description": "Open a terminal window with Ctrl-RAlt-T",
+        "manipulators": [
+            {
+                "from": {
+                    "key_code": "t",
+                    "modifiers": {
+                        "mandatory": [
+                            "left_control",
+                            "right_option"
+                        ],
+                        "optional": [
+                            "any"
+                        ]
```

```suggestion
                        "optional": [
                            "caps_lock"
                        ]
```

Using `any` in `optional` modifiers will execute this manipulator for all `t+<modifiers>` combo. Either don't use any `optional` modifiers, or only add `caps_lock`, so this will work when caps lock is on.

---

Perfect, sounds like an improvement indeed, thank you!